test/EzMap.IntegrationTest/PoiControllerTest.cs

```diff
@@ -99,4 +99,52 @@ public async Task DeletePoi_CorrectDataProvided_PoiShouldBeDeleted()
         Assert.Equal(HttpStatusCode.OK, response.StatusCode);
         // assert
     }
+
+    [Fact]
+    public async Task GetPoi_GuidProvided_DetailOfPoiShouldBeFound()
+    {
+        var app = new TestWebAppFactory<Program>();
+        var client = app.CreateClient();
+        using var scope = app.Services.CreateScope();
+        var dbContext = scope.ServiceProvider.GetRequiredService<EzMapContext>();
+        var token = await TestHelper.GetDefaultUserToken(client);
+
+
+        var user = new User("thanh", "thanh", "thanh", "thanh");
+        dbContext.Users.Add(user);
+        var poi = new Poi("home", "59 ntt", user.Id);
+        dbContext.Pois.Add(poi);
+
+        await dbContext.SaveChangesAsync();
+        
+        // act
+        using var response = await client.RequestAsJsonAsyncWithToken<object>(HttpMethod.Get, $"api/poi/{poi.Id}", token);
+        response.EnsureSuccessStatusCode();
+        Assert.Equal(HttpStatusCode.OK, response.StatusCode);
```

Comment:
This assert is inadequate, should assert all other properties of the returned Poi as well. Remember, Success status doesn't indicate Logic success